lib/tokenizer/index.js

```diff
@@ -1413,7 +1413,10 @@ _[BEFORE_ATTRIBUTE_VALUE_STATE] = function beforeAttributeValueState(cp) {
 
     else if (cp === $.APOSTROPHE)
         this.state = ATTRIBUTE_VALUE_SINGLE_QUOTED_STATE;
-
+    else if (cp === $.GREATER_THAN_SIGN) {
+        this._err(ERR.missingAttributeValue);
+        this._reconsumeInState(ATTRIBUTE_VALUE_UNQUOTED_STATE);
```

Comment:
Missing blank line before and after new `else if` block. Let's rewrite it as per our spec changes: switch to data state here and emit current token instead of reconsumption.